### PR TITLE
Add consistent stroke color for actors in sequence diagram

### PR DESCRIPTION
**Pull request by Shairam**
cell-level

> Service - Level Diagram
service-level

### CODE REVIEW

File: system/control-plane/global/components/orbit/grpc/dependency-reduced-pom.xml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (sinthuja) [inline system/control-plane/global/components/orbit/grpc/dependency-reduced-pom.xml]: Why this pom.xml is added? Also this pom.xml is missing the license header.

File: system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/tracing/utils/constants.js [hunk 2]
```diff
@@ -1,48 +0,0 @@
-/*
```
Comment (sinthuja) [inline system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/tracing/utils/constants.js]: Why this is removed?

File: system/control-plane/global/components/widgets/OpenTracingVisTimeline/package-lock.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
       "resolved": "https://registry.npmjs.org/@wso2-dashboards/widget/-/widget-1.2.2.tgz",
       "integrity": "sha512-j1sg6tgHonBcjeSh/DRD7nPdzEQOto18FBogbizzq2q+Y6uUx63+rqCvJl2rICQbJaxvsGLW1RrT3odDu7TZ0Q==",
       "requires": {
-        "axios": "0.17.1"
```
Comment (sinthuja) [inline system/control-plane/global/components/widgets/OpenTracingVisTimeline/package-lock.json]: These widgets are not used, can you remove the changes for the package-lock.json from widgets?

File: system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/tracing/utils/constants.js (hunk 2 above)
Comment (Shairam) [reply system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/tracing/utils/constants.js]: We use the constants from [https://github.com/Shairam/product-vick/blob/master/system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/common/constants.js](https://github.com/Shairam/product-vick/blob/master/system/control-plane/global/components/observability/org.wso2.vick.observability.ui/src/pages/common/constants.js)

**Pull request merged**